src/main/java/io/reactivex/Completable.java

```diff
@@ -1721,9 +1743,7 @@ public final Disposable subscribe(final Action onComplete, final Consumer<? supe
     public final Disposable subscribe(final Action onComplete) {
         ObjectHelper.requireNonNull(onComplete, "onComplete is null");
 
-        CallbackCompletableObserver s = new CallbackCompletableObserver(onComplete);
-        subscribe(s);
-        return s;
+        return subscribe(onComplete, Functions.ON_ERROR_MISSING);
```

Comment:
No need for null checks. Also since that subscribe delegates to a third method, one call indirection could be avoided by call the new method directly.